server/controllers/userController.js

```diff
@@ -188,6 +189,34 @@ class UsersController {
     }
   }
 
+  static async updateUser(req, res) {
+    try {
+      const { email, first_name, last_name, dob, year_of_graduation, phone_number, role } = res.locals.user;
+      const loggedInUser = req.session.role;
```

Comment:
Can you make the variable `loggedInUser` more descriptive? We can append `Role` to it to make it `loggedInUserRole`.